### DISCUSSION TITLE
Select keybinding ignores hold-to-confirm activation time

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19923] [Q&A] [answered]
I don't think this should work like that.

https://user-images.githubusercontent.com/86934170/186102131-ede5de8a-6ab0-400a-b804-cea78a43ee65.mp4

**Comment by peppy** [comment, marked as answer]
This was intentional during implementation, but I'm open to changing it.

**Reply by Loreos7** [reply]
I agree that it should be instant using space bar. It's like another way to act with the game if you have to delete many beatmaps.
